dev/webpack.config.js

```diff
@@ -0,0 +1,161 @@
+/* eslint import/no-commonjs: 0 */
+const path = require('path');
+const without = require('lodash/without');
+
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackPluging = require('html-webpack-plugin');
+const BabiliPlugin = require('babili-webpack-plugin');
+const AutoDllPlugin = require('autodll-webpack-plugin');
+const HappyPack = require('happypack');
+
+const __DEV__ = process.env.NODE_ENV === 'development';
+const clean = arr => arr.filter(item => item !== false);
+
+module.exports = {
```

Comment:
imho defining three configurations: `common`, `dev`, `prod` is more legible and easy to follow.